assembly/main.ts

```diff
@@ -56,6 +57,7 @@ export function transferFrom(from: string, to: string, tokens: u64): boolean {
   const approvedAmount = allowance(from, to);
   assert(tokens <= approvedAmount, "not enough tokens approved to transfer");
   balances.set(from, fromAmount - tokens);
+  assert(getBalance(to) <= getBalance(to) + tokens,"overflow at the receiver side");
```

Comment:
Perhaps do this first before the balance is set.

Reply:
Yeah sure, that would perhaps save some computation. (Fail Early Fail Loud :))

Reply:
Thanks!